AcadeMeData/tests.py

```diff
@@ -47,51 +35,116 @@ def test_get_user(self):
-                                                   name="DR Arnold Schwarzenegger",
-                                                   university=university,
-                                                   description="A cool guy who looked familiar",
-                                                   rate=4.5)
-        assert professor_for_example.get_name() == "DR Arnold Schwarzenegger"
-
-    def test_create_professor(self, name="DR Arnold Schwarteneiger"):
-        professor_for_example = self.get_proffesor()
-        professor = Professor.get_proffesor(name)
+    def test_get_name(self, professor_id=1, name="DR Arnold Schwarzenegger", university=None,
+                      description="A cool guy who looked familliar", rate=4.5, university_id=1,
+                      uni_name='Reichman University', location="Herzlia",
+                      uni_description="A nice place"):
+        university = TestUniversityModel.generate_university(self, university_id, uni_name, location, uni_description)
+        professor_for_example = self.generate_professor(professor_id, name, university, description, rate)
+        assert professor_for_example.get_name() == name
+
+    def test_create_professor(self, professor_id=1, name="DR Arnold Schwarzenegger", university=None,
+                              description="A cool guy who looked familliar", rate=4.5, university_id=1,
+                              uni_name='Reichman University', location="Herzlia",
+                              uni_description="A nice place"):
+        university = TestUniversityModel.generate_university(self, university_id, uni_name, location, uni_description)
+        professor_for_example = self.generate_professor(professor_id, name, university, description, rate)
+        professor = Professor.get_professor(name)
         assert professor.get_name() == professor_for_example.name
         assert professor.get_description() == professor_for_example.description
+
+
+@pytest.mark.django_db
+class TestMessageBoardModel:
+    def generate_msgboard(self, id=1, courseName="Linear Algebra"):
+        msgboard = MessageBoards(id=id, courseName=courseName)
+        msgboard.save()
+        return msgboard
+
+    def test_get_msgboard(self, id=1, courseName="Linear Algebra"):
+        msgboard = TestMessageBoardModel.generate_msgboard(self, id, courseName)
+        msgboard_test = MessageBoards.get_msgboard_by_id(1)
+        assert msgboard_test == msgboard
+        assert isinstance(msgboard_test, MessageBoards)
+
+
+@pytest.mark.django_db
+class TestMessagesModel:
+    def user_example(self):
+        user_data = {'username': "username22", 'password': "password", 'email': "user@example.com", 'type': "S",
+                     'university': "RU",
+                     'degree': "CS"}
+        user = User.create_user(*user_data)
+        return user
+
+    def generate_message(self, msgID=1, userID=None, text='This is a test message yo'):

```

Comment:
Please start to use @fixtures for populating the content for tests

Reply:
> Please split the tests to different files. in the same directory

How can I do that? in the lecture we didn't talk about splitting the tests to different files

Reply:
fixtures are covered in the last class

Reply:
Yariv checkout the tests, I added fixtures. 
The splitting to apps will be done after, with the help of the latest PR from dan (he already divided everything)

Reply:
OK, splitting is too complex, lets leave it to discussion on PR #69

Reply:
missing fixtures